ts/Core/Foundation.ts

```diff
@@ -83,7 +83,9 @@ namespace Foundation {
 
                     if (isFunction(event)) {
                         component.eventOptions[eventType] = event;
-                        addEvent(component, eventType, event);
+                        addEvent(component, eventType, event, {
+                            order: 0 // #14080 fire those events as firsts
```

Comment:
I'm not sure I understand this correctly - but wouldn't it be better to set a higher order to the event in `BubbleLegendComposition.ts` instead of updating the core functionality?

Reply:
That was my first idea: change the order of that event in `BubbleLegendComposition.ts`, but unfortunately, it is not working. Events, which are added by users from chart configuration are added with order equal to `Infinity` - that's why, even if I add some order to an event in `BubbleLegendComposition.ts` it won't work, because there is no number bigger than `Infinity`... As a consequence of that, I decided to add `order: 0` to all "user's event functions" and it solves this problem. I think, that there may be more such problems, maybe not reported by users, or in the future, if someone would like to add an event, which will be connected also to events, which are added by users from chart configuration.

Reply:
I assume we chose to set `Infinity` to default handlers so that any user-defined handlers could prevent the default, right? So now with this change, we still have Infinity on _default handlers_, but we have `order: 0` on any event that is defined in the _chart configuration_. Is that correctly understood? If so, I think it makes sense.

Reply:
Yes, that's true. 
Default handlers - `Infinity`to able to prevent. 
Events defined in the chart configuration -`order: 0` to fire them as firsts.